tests/linux/2024.vncjew/__min__c.simple

```diff
@@ -1,13 +1,71 @@
-# linux/2024.vncjew/__min__c: critical
-anti-static/elf/entropy: high
-anti-static/elf/header: high
-anti-static/elf/multiple: medium
-anti-static/packer/upx: high
-c2/addr/ip: high
+# linux/2024.vncjew/__min__c ∴ /__min__c: critical
+c2/addr/ip: medium
 c2/addr/url: low
 c2/tool_transfer/arch: low
-fs/proc/self_exe: medium
+credential/password: low
+credential/ssl/private_key: low
+crypto/aes: low
+crypto/cipher: medium
+crypto/decrypt: low
+crypto/ecdsa: low
+crypto/ed25519: low
+crypto/encrypt: medium
+crypto/public_key: low
+crypto/rc4: low
+crypto/tls: low
+data/compression/gzip: low
+data/encoding/base64: low
+data/encoding/json: low
+data/encoding/json_decode: low
+data/hash/md5: low
+discover/system/cpu: low
+discover/system/hostname: low
+discover/system/platform: low
+discover/user/HOME: low
+discover/user/USER: low
+evasion/bypass_security/linux/iptables: medium
+evasion/bypass_security/linux/iptables_append: medium
+exec/plugin: low
+exec/program: medium
+fs/directory/list: low
+fs/file/open: low
+fs/file/read: low
+fs/link_read: low
+fs/path/etc: low
+fs/path/etc_hosts: medium
+fs/path/etc_resolv.conf: low
+fs/path/home: low
+fs/permission/chown: medium
+fs/permission/modify: medium
+fs/tempfile: low
+malware/family/vncjew: critical
+net/dns: low
+net/dns/servers: low
 net/dns/txt: low
+net/http/accept: low
+net/http/accept_encoding: low
+net/http/auth: low
+net/http/cookies: medium
+net/http/post: medium
+net/http/proxy: low
 net/http/request: low
-net/remote_control/vnc: medium
```

Comment:
this change was surprising - I feel like this should have fired in both forms.

Reply:
Updated in [`f849ff6` (#731)](https://github.com/chainguard-dev/malcontent/pull/731/commits/f849ff659f7acd25b7a2f71e2b643ea13b027088#diff-98d64e984a50bac5a4a8314d16140f66b05e118db774dc53208d9896d61352f9).